mappings/net/minecraft/world/level/ColorResolver.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/world/level/ColorResolver
+	METHOD getColor (Lnet/minecraft/class_1959;DD)I
+		ARG 2 xColor
+		ARG 4 zColor
```

Comment:
I don't understand. This method takes two colors and outputs a new color?

Reply:
Or should this be `colorX` and `colorZ` (the coordinates of the color, not colors at the coordinates)? In that case I think just `x`, `z` would be enough.

Reply:
I think this one just gets a color from the x z values.

Reply:
@liach  you are correct they are the x and z colors at the z and z pos

Reply:
https://github.com/FabricMC/yarn/pull/1218/commits/473d1db5a9f7eafdade03de43f570152716a64ca